css-ecommerce-page/index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Ecommerce Challenge</title>
+  <link href="https://fonts.googleapis.com/css?family=DM+Sans&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="header">
+    <header>
+      <p>New Arrivals</p>
```

Comment:
Semantically, this should be a heading (`<h#>`), not a paragraph (`<p>`). Also, it needs a higher font-weight:

---

![image](https://user-images.githubusercontent.com/16143771/152213314-da826d3f-6ba6-4214-8a35-24bd7bb0a48f.png)

---

![image](https://user-images.githubusercontent.com/16143771/152213334-34e0f6f2-3f0f-479b-8c37-8fd862df8a3e.png)